### ISSUE TITLE
Comments on lines 68-75

### ISSUE THREAD

**Issue by VLucet** [pedersen-fisheries-lab/sspm#115]

**Comment by VLucet** [comment]
> 68-75- This paragraph makes it seem like the package creation was an after thought instead of the goal being to design an R package that can accommodate spatial data while fitting SSPMs. "and apply to any spatially structured stock" is grammatically incorrect. It seems like the sentence starting on line 62 through the end of that paragraph pertains more to this paragraph than where it is currently located. I would change where the paragraphs are separated and make sure that information is not redundant.

**Comment by VLucet** [comment]
@eric-pedersen I'd like your input on item 1 in this list

**Comment by eric-pedersen** [comment]
I'd suggest rewording:

"The R package sspm is designed to make spatially-explicit surplus production models (SSPM) simpler to estimate and apply to any spatially structured stock. The basic model it implements was first used to model time-varying production in Newfoundland and Labrador Northern Shrimp stocks [@pedersenNewSpatialEcosystembased2021]. However, the general modelling approach used here will work for any spatially structured fishery with sufficient data."


Also: we can update the reference for the res doc; it's no longer in press! https://www.dfo-mpo.gc.ca/csas-sccs/Publications/ResDocs-DocRech/2022/2022_062-eng.html. The year for the res doc is 2022, and the document number is 2022/062.

**Comment by VLucet** [comment]
I have reworded. I will update the ref accordingly as well.
